wrapper.go

```diff
@@ -200,8 +202,21 @@ func makeV2Request(event Request, prefix string) (*http.Request, error) {
 		return nil, fmt.Errorf("unable to create request: %w", err)
 	}
 
+	var contentLength int64
+	if s, ok := event.Headers["content-length"]; ok {
+		v, err := strconv.ParseInt(s, 10, 64)
+		if err != nil {
+			return nil, fmt.Errorf("unable to parse content length, %v: %w", s, err)
+		}
+		contentLength = v
+	}
+
 	setHeader(event, req)
 
+	req.ContentLength = contentLength
```

Comment:
not sure if you wanted these properties unset on v2 payloads, I can revert this if needed, was just filling out a missing test so I could test the header stuff.